CHANGELOG.md

```diff
@@ -70,20 +70,178 @@
+  [#7765](https://github.com/Kong/kong/pull/7765)
+
+#### Performance
+
+On this release we've done some special efforts with regards to performance.
+
+There's a new performance workflow which periodically checks new code additions against some
+typical scenarios [#7030](https://github.com/Kong/kong/pull/7030) [#7547](https://github.com/Kong/kong/pull/7547)
+
+In addition to that, the following changes were specifically included to improve performance:
+
+- Reduced unnecessary reads of `ngx.var`
+  [#7840](https://github.com/Kong/kong/pull/7840)
+- Loaded more indexed variables
+  [#7849](https://github.com/Kong/kong/pull/7849)
+- Optimized table creation in Balancer
+  [#7852](https://github.com/Kong/kong/pull/7852)
+- Reduce calls to `ngx.update_time`
+  [#7853](https://github.com/Kong/kong/pull/7853)
+- Use read-only replica for PostgreSQL meta-schema reading
+  [#7454](https://github.com/Kong/kong/pull/7454)
+- URL escaping detects cases when it's not needed and early-exists
+  [#7742](https://github.com/Kong/kong/pull/7742)
+- Accelerated variable loading via indexes
+  [#7818](https://github.com/Kong/kong/pull/7818)
+
+#### Configuration
+
+- Enable IPV6 on `dns_order` as unsupported experimental feature
+  [#7819](https://github.com/Kong/kong/pull/7819).
+- The template renderer can now use `os.getenv`
+  [#6872](https://github.com/Kong/kong/pull/6872).
+
+#### Hybrid Mode
+
+- Data plane is able to eliminate some unknown fields when Control Plane is using a more modern version
+  [#7827](https://github.com/Kong/kong/pull/7827).
+
+#### Admin API
+
+- Add support for HEAD requests
+  [#7796](https://github.com/Kong/kong/pull/7796)
+- Improve support for OPTIONS requests
+  [#7830](https://github.com/Kong/kong/pull/7830)
 
 #### Plugins
 
-- **aws-lambda**: The plugin will now try to detect the AWS region by using `AWS_REGION` and
+- **AWS-Lambda**: The plugin will now try to detect the AWS region by using `AWS_REGION` and
   `AWS_DEFAULT_REGION` environment variables (when not specified with the plugin configuration).
   [#7765](https://github.com/Kong/kong/pull/7765)
+- **Datadog**: `host` and `port` config options can be configured from `ENV` variables
+  [#7463](https://github.com/Kong/kong/pull/7463)
+  Thanks [rallyben](https://github.com/rallyben) for the patch!
+- **Prometheus**: The new `data_plane_cluster_cert_expiry_timestamp` exposes DP cert expiry timestamp
+  [#7800](https://github.com/Kong/kong/pull/7800).
+
+**GRPC-Gateway**:
+
+- The plugin can decode entities of type `.google.protobuf.Timestamp`
+  [#7538](https://github.com/Kong/kong/pull/7538)
+- Added support for structured URI arguments
+  [#7564](https://github.com/Kong/kong/pull/7564)
+  Thanks [git-torrent](https://github.com/git-torrent) for the patch!
+
+**Request Termination**:
+
+- New `trigger` config option, which makes the plugin activate for any requests with a header or query parameter
+  named like the trigger
```

Comment:
```suggestion
- New `trigger` config option, which makes the plugin only activate for any requests with a header or query parameter
  named like the trigger. This can be a great debugging aid, without impacting actual traffic being processed.
```